Handle bugs for filtering without characteristics (#181)
AY2324S1-CS2103T-W11-2/tp: docs/UserGuide.md

Format: `filtercust [b/BUDGET] [c/CHARACTERISTIC]…​`

> * Filter and return the customer(s) whose budget is bigger than or equal with the `BUDGET` and who is/are seeking for properties which have all the `CHARACTERISTIC`.

Reviewer: ```suggestion
* Filter and return the customers whose budget is bigger than or equals to the `BUDGET` and are seeking for properties which have all the `CHARACTERISTIC`.
```
might be neater this way?

Filters the property list to only show properties that fit the criteria.

Format: `filterprop [pr/PRICE] [c/CHARACTERISTIC]…​`

> * Filter and return property or properties whose price is lower than or equal with the `pr/PRICE` and which has or have all the `c/CHARACTERISTIC`.

Reviewer: ```suggestion
* Filter and return properties whose price is lower than or equals to the `PRICE` and have all the `CHARACTERISTIC`.
```
would be good to standardise the parameters as above!

* `pr/PRICE` (optional)          : The price of the property
* `c/CHARACTERISTIC` (optional)  : The characteristics of the property
* Even though both `PROPERTY` and `CHARACTERISTIC` are optional, at least one of them should exist.

> * Not giving the `PRICE` will return the customer(s) who is/are seeking for properties which have all the `CHARACTERISTIC`.

Reviewer: should be properties?

> * Not giving the `CHARACTERISTIC` will return the customer(s) whose price is lower than or equal with `PRICE`.

Reviewer: likewise for this!
